Cargo.toml

```diff
@@ -47,6 +56,15 @@ codegen_spec = { path = "crates/codegen/spec" }
 codegen_testing = { path = "crates/codegen/testing" }
 infra_cli = { path = "crates/infra/cli" }
 infra_utils = { path = "crates/infra/utils" }
+
+slang_testlang = { path = "crates/testlang/outputs/cargo/crate" }
```

Comment:
I think this might need coordination with the changes from #745, whichever lands sooner. cc @Xanewok

Reply:
Done